main/vstring.c

```diff
@@ -81,12 +81,12 @@ extern vString *vStringNew (void)
 
 extern void vStringPut (vString *const string, const int c)
```

Comment:
> As I mentioned from my experience the compiler will ignore it in any case.

With which compiler? I think GCC always tries its best to inline functions marked with `inline`.